packages/core/lib/modules/wallet.js

```diff
@@ -27,6 +29,19 @@ export const WalletEvents = {
   networkConnected: 'network-connected',
 };
 
+/**
```

Comment:
what if instead of passing a bool we passed the path to the schema file that we want to load? Would support different schema versions that way as we go forward.

Reply:
Ok, we can leave it for now but would be good to support different versions easily.